packages/cli/src/ceramic-daemon.ts

```diff
@@ -295,14 +297,19 @@ export class CeramicDaemon {
     const ceramic = new Ceramic(modules, params)
     if (opts.stateStore?.mode == StateStoreMode.S3) {
       const s3Store = new S3Store(opts.stateStore?.s3Bucket, params.networkOptions.name)
-      await ceramic.repository.injectStateStore(s3Store)
+      await ceramic.repository.injectKeyValueStore(s3Store)
     }
     const did = new DID({ resolver: makeResolvers(ceramic, ceramicConfig, opts) })
     ceramic.did = did
     await ceramic._init(true)
 
     const daemon = new CeramicDaemon(ceramic, opts)
     await daemon.listen()
+    daemon.anchorResumingService
```

Comment:
I'm curious, is there a reason that the anchorResumingService is a part of the CeramicDaemon instead of the Ceramic core client?

Reply:
I did it this way, because every time I've been a part of discussing how this should work, we were saying that the resume should happen when the daemon is restarted.

Happy to move it to Ceramic core, if you think it should happen always when the core is client is started. I assume this should happen in `Ceramic._init(...)`, right?

Reply:
That's right @pawartur. Agree that this makes more sense in core, because if you are using core directly this feature is still desirable.